src/core/queryClient.ts

```diff
@@ -123,6 +123,31 @@ export class QueryClient {
       .setData(updater, options)
   }
 
+  setQueriesData<TData>(
+    queryKey: QueryKey,
+    updater: Updater<TData | undefined, TData>,
+    options?: SetDataOptions
+  ): [QueryKey, TData][]
+
+  setQueriesData<TData>(
+    filters: QueryFilters,
```

Comment:
I didn't create an overload that accepts queryKey _and_ filters, even though some other functions have this, because the `updater` is required so the signature would be very weird. I also don't think it's important because the `filters` also have the option to pass a `queryKey`.